pkg/build/api/types.go

```diff
@@ -707,6 +739,12 @@ type BuildRequest struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// SkipBuildsInQueue will cause all builds in the queue marked as skipped and
+	// make the build created from this build requests be the only build in the
+	// queue. This option is valid only for build configurations with "recent"
+	// build scheduling policy.
+	SkipBuildsInQueue *bool
```

Comment:
There isn't a direct equivalent afaik. There's also accelerated build which will run the new build immediately, but not cancel the other builds. It's all possible of course :) @jstrachan Do you know any different?